### ISSUE TITLE
Use native icons

### ISSUE THREAD

**Issue by ruizlenato** [l4l/yofi#76]
Some launchers take the icons from the app itself when the icon pack does not have the app icon, it would be nice to have this in yofi.

![image](https://user-images.githubusercontent.com/49354525/151427577-e10fab20-9539-4996-8af5-d321175a8306.png)
> Sirula

![image](https://user-images.githubusercontent.com/49354525/151427705-1c0a1cd3-5e40-450f-b9c3-419ccb43d66f.png)
> yofi

**Comment by l4l** [comment]
I suppose that should be resolved after #13 which actually requires quite a significant rework of the logic.

Currently yofi does grab apps' names/paths of the icon from .desktop files (provided by the app) thus it should work most of the times. The problem that current implementation could only handle a single size of icon. I.e. the flow is the following:

1. Find `Icon=` entry in desktop file (e.g. /usr/share/applications/firefox.desktop).
2. If that doesn't look <s>and quack</s> like path then find a specified name in `/usr/share/icons/<icon_theme>/<icon_size>x<icon_size>` (and similar).
3. Load the icon by the final path.

The problem is that `<icon_size>` is fixed and if an app has a different icon_size it won't be found. So for temporary solution one of the (poor) workarounds might work out for you:

- Change icon.size in [config](https://github.com/l4l/yofi/blob/master/README.md#configuring).
- Specify absolute icon path for certain desktop files.
- Create rescaled icon and put it at `/usr/share/icons/<icon_theme>/<icon_size>x<icon_size>` (<icon_theme> by default is `default`).
